model/chunks/chunkFaults.go

```diff
@@ -79,6 +79,35 @@ func NewCFNonMatchingFinalState(expected flow.StateCommitment, computed flow.Sta
 		execResID:  execResID}
 }
 
+// CFInvalidEventsCollection is returned when computed events collection hash is different from the chunk's one
+type CFInvalidEventsCollection struct {
+	expected   flow.Identifier
+	computed   flow.Identifier
+	chunkIndex uint64
+	execResID  flow.Identifier
```

Comment:
```suggestion
	resultID flow.Identifier
```
Just to be consistent with the rest of codebase